src/e2e-test/java/io/cdap/plugin/utils/DataStoreClient.java

```diff
@@ -0,0 +1,154 @@
+package io.cdap.plugin.utils;
+
+import com.esotericsoftware.minlog.Log;
+import com.google.cloud.datastore.Datastore;
+import com.google.cloud.datastore.DatastoreOptions;
+import com.google.cloud.datastore.Entity;
+import com.google.cloud.datastore.FullEntity;
+import com.google.cloud.datastore.IncompleteKey;
+import com.google.cloud.datastore.Key;
+import com.google.cloud.datastore.KeyFactory;
+import com.google.cloud.datastore.Query;
+import com.google.cloud.datastore.QueryResults;
+import com.google.gson.Gson;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import io.cdap.e2e.utils.PluginPropertyUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Represents DataStore client.
+ */
+public class DataStoreClient {
+  static Gson gson = new Gson();
+  private static final Logger logger = LoggerFactory.getLogger(PubSubClient.class);
+  static Datastore datastore = DatastoreOptions.newBuilder().
+                               setProjectId(PluginPropertyUtils.pluginProp("projectId")).build().getService();
+  static Key key;
```

Comment:
why is this static?

Reply:
A global instance variable does not need to be static and can be initialized in constructors.

Why are we keeping them static is still not answered?

Reply:
I see that all methods are also static in other clients.